met-api/migrations/versions/8ca063aafc01_create_comment_table.py

```diff
@@ -0,0 +1,56 @@
+"""empty message
+
+Revision ID: 8ca063aafc01
+Revises: 6d3c33a79c5e
+Create Date: 2022-07-28 15:13:35.368696
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from datetime import datetime
+
+
+# revision identifiers, used by Alembic.
+revision = '8ca063aafc01'
+down_revision = '6d3c33a79c5e'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    comment_status_table = op.create_table('comment_status',
+    sa.Column('id', sa.Integer(), autoincrement=True, nullable=False),
```

Comment:
I think you missed removing the autoincrement=True

Reply:
removed, good eye